.github/workflows/e2e.yml

```diff
@@ -19,20 +19,58 @@ on:
         description: 'Alternative Password'
         required: false
         default: ''
+
+permissions: read-all
```

Comment:
Good point, will do that now.

Reply:
I have set the workflow permissions to `contents: read`. This seems to be more restrictive than `permissions: read-all`, because setting only one permission [behaves as if all the other permissions were set to `none`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions).